bin/crew

```diff
@@ -676,6 +677,10 @@ def build_and_preconfigure(target_dir)
     puts 'Preconfiguring package...'
     cache_build if CREW_CACHE_BUILD
     @pkg.install
+    build_end_time = Time.now.to_i
+    build_time_elapsed = build_end_time - build_start_time
+    crewlog "Build & install time for #{@pkg.name} was #{build_time_elapsed} seconds."
+    puts "Build & install time for #{@pkg.name} was #{build_time_elapsed} seconds. Please ask for #{ARCH} binaries to be generated for #{@pkg.name}.".lightpurple if build_time_elapsed > 60
```

Comment:
Include hours and minutes in the time?  Some builds can require all night and it will be hard to determine the actual time without converting.

Reply:
Added hours and minutes...